benchmarks/pipe.py

```diff
@@ -297,10 +307,13 @@ def train(data_config, model, benchmark_config, args):
                 total_loss = 0
                 start_time = time.time()
 
+    return total_words, loss.item()
+
 
-def evaluate(eval_model, data_source, criterion, bptt, ntokens):
+def evaluate(eval_model, data_source, criterion, ntokens):
     eval_model.eval()
     total_loss = 0.0
+    bptt = 35  # Used for evaluation not training.
```

Comment:
the comment seems to be obvious. It would be useful to explain why is it 35, how is that chosen if we know it.

Reply:
I am not aware of why we chose it. We don't benchmark evaluation right now. I've added a comment to followup though.